verify/builtin/lists/vatomic32.txt

```diff
@@ -0,0 +1,195 @@
+vatomic32_add
```

Comment:
did you automatically generate these lists? do they contain all 32bit functions?